esx_service/vmodl/vmodl_test.py

```diff
@@ -690,7 +690,7 @@ def test_update_privilege_with_invalid_volume_size(self):
 
         # Update the privilege with invalid volume size
         with self.assertRaises(vmodl.fault.InvalidArgument):
-            self.tenantMgr.UpdatePrivilege(tenant, self.datastore, volume_max_size=2048, volume_total_size=1024)
+            self.tenantMgr.UpdatePrivilege(tenant, self.datastore, allow_create=True, volume_max_size=2048, volume_total_size=1024)
```

Comment:
This is incorrect. The auth_api._tenant_access_set() method should be able to handle the case where the allow_create parameter is not set (i.e. set to None). This API is meant to handle update request - it's fair enough that user may just need to update some other properties, but leave existing allow_create unchanged.